libindy/src/commands/signus.rs

```diff
@@ -205,11 +215,28 @@ impl SignusCommandExecutor {
                 CommonError::InvalidState(
                     format!("Can't serialize MyDid: {}", err.description())))?;
 
-        self.wallet_service.set(wallet_handle, &format!("my_did::{}", my_did.did), &my_did_json)?;
+        self.wallet_service.set(wallet_handle, &format!("my_did_temporary_keys::{}", my_did.did), &my_did_json)?;
 
         Ok((my_did.verkey, my_did.pk))
     }
 
+    fn replace_keys_apply(&self,
+                          wallet_handle: i32,
+                          did: &str,
+                          cb: Box<Fn(Result<(), IndyError>) + Send>) {
+        cb(self._replace_keys_apply(wallet_handle, did));
+    }
+
+    fn _replace_keys_apply(&self,
```

Comment:
Should we delete temporary entry?
@vimmerru

Reply:
It is better to remove from my point of view